style_homepage.css

```diff
@@ -0,0 +1,774 @@
+        height: 150px;
+        flex-direction: column;
+    }
+    .flex_container_work
+    {
+    display:flex;
+    width:60%;
+    flex-direction: column;
+    border: 1px black solid;
+    }
+.flex_container_work_01, .flex_container_work_02
+    {
+    display: flex;
+    height:50%;
+    flex-direction: column;
+    }
+}
+
+/*Contact page styling*/
+
+.container /*Contact form main container*/
+{
+    padding-top: 5%;
+    width: 80%;
+    padding: 10%;
+    background-color:#db9cc0;
+}
+
+.contact-box
+{
+    background:#fffdf0;
+    width: 100%;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.contact-left{
+    flex-basis: 70%;
+    padding: 7% 0;
+}
+
+.input-row{
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 20px;
+
+}
+.input-row .input-group{
+    flex-basis: 45%;
+}
+input{
+    width: 100%;
+    border: none;
+    border-bottom: 1px solid #ccc;
+    outline: none;
+    padding: 5%;
+}
+textarea{
+    width: 100%;
+    border: 1px solid #ccc;
+    outline: none;
+    padding: 10px;
+    box-sizing: border-box;
+}
+
+label{
+
+    margin-bottom: 6px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1rem;
+    display: block;
+    color: #af7c99;
+}
+
+button{
+background:#db9cc0;
+width: 8rem;
+border: none;
+outline: none;
+color: #fff;
+height: 3rem;
+border-radius: 30px;
+margin-top: 20px;
+box-shadow: 0px 5px 15px 0px rba(28,0,181,0.3);
+}
+
+.contact-left h3{
+    color: #000;
+    font-weight: 600;
+    margin-bottom: 30px;    
+
+}
+
+/*Project page styling*/
+
+.flex_container_project
+{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: auto;
+}
+
+.flex_container_project_01
+{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    background-color: #db9cc0;
+    
+}
+
+.project_01_first_div
+{
+    color: #fffdf0;
+    font-size: 2.5rem;
+    text-align: center;
+    width: 80%;
+    padding-top: 10%;
+    padding-left: 9%;
+}
+
+.project_01_second_div
+{
+    background-color:#db9cc0;
+    width: 100%;
+    padding-left: 20%;
+    padding-bottom: 5%;
+}
+
+.project_01_second_div img
+{
+    max-width: 60%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.flex_container_project_02
+{
+    display: flex;
+    width: 100%;
+    height: 70%;
+    font-size: 1.5rem;
+    color: #fffdf0;
+    background-color: black;
+    justify-content: center;
+}
+
+.flex_container_project_03
+{
+    display: flex;
+    width: 100%;
+    background-color:black;
+    text-align: center;
+    padding-bottom: 5%;
+}
+
+.flex_container_project_03_first_div
+{
+    width: 100%;
+    text-align: center;
+}
+
+.flex_container_project_03_first_div img
+{
+    max-width: 90%;
+}
+
+.flex_container_project_03_first_div img:hover
+{
+    box-shadow: 0 0 2px 1px rgba(0, 140, 186, 0.5);
+    filter:grayscale(100%)
+}
+.project_01_second_div img:hover
+{
+    box-shadow: 0 0 2px 1px rgba(0, 140, 186, 0.5);
+    filter:grayscale(100%)
+} 

```

Comment:
These classes are named with 01, 02, 03, and very similar names like "flex container" at the beginning of every one. This makes it very hard to read the HTML and compare it to the CSS, and so it makes it harder for you to debug when there are issues. Next time try writing more expressive names, like `gallery` and `gallery-item`. If you want to differentiate the small and big galleries, you could have `big-gallery`, `big-gallery-item`, `small-gallery`, `small-gallery-item`.